### ISSUE TITLE
Jump to navigation accessibility update

### ISSUE THREAD

**Issue by cathystoli** [chanzuckerberg/sci-components#699]
**Issue:**
1. Cannot use component with keyboard navigation alone (works with VoiceOver.)

2. No focus indicators.

3. When utilizing component with VoiceOver, if user is at the top of the page and tries to select Section 2, it navigates to section 2 but the focus indicator from VoiceOver does not jump correctly and it ends up not being focused on the right item.

4. Utilizes arrow keys to go to next section within component. Should it use arrow keys or tab key?

**Solution:**
1. Allow component to be used via keyboard navigation.

2. Add focus indicators.

3. Fix behavior so that the focus indicator functions correctly when jumping from the top of the page.

4. OQ: Does it make more sense to use arrow keys or tab to go to the next section within the component?
